inc/class-shortcode-ui.php

```diff
@@ -199,12 +207,17 @@ public function action_admin_enqueue_scripts( $editor_supports ) {
 
 		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
 
-		wp_register_script( 'select2',
+		wp_register_script( self::$select2_handle,
 			trailingslashit( $this->plugin_url ) . "lib/select2/js/select2.full{$min}.js",
 			array( 'jquery', 'jquery-ui-sortable' ), '4.0.3'
 		);
 
-		wp_register_style( 'select2',
+		if( self::$select2_handle !== 'select2' ){
+			 wp_add_inline_script( self::$select2_handle, 'var existingSelect2 = jQuery.fn.select2 || null; if (existingSelect2) { delete jQuery.fn.select2; }', 'before' );
+			 wp_add_inline_script( self::$select2_handle, 'jQuery.fn.'.self::$select2_handle.' = jQuery.fn.select2; if (existingSelect2) { delete jQuery.fn.select2; jQuery.fn.select2 = existingSelect2; }', 'after' );
```

Comment:
> With that the escaping should be automatically done, right?

Yeah, I think the escaping that's being done to prepare the localization strings for json encoding is safe enough. It may not actually guarantee a valid attribute name, but it will prevent passing something like `);alert('xss')` in and running that arbitrary script, which was my concern.